### PR TITLE
Add online get started examples for F#

### LINKED ISSUE #2554: Write something other than links under Get Started index page
It's currently serving no practical purpose other than linking to child content.  This should either:

* Contain more words about getting started with F#, possibly linking to Azure Notebooks and Azure Functions
* Be empty and  link directly to Get Started in VS

### PR DESCRIPTION
**Pull request by cartermp**
Fixes #2554

### CODE REVIEW

File: docs/fsharp/get-started/index.md [hunk 1]
```diff
@@ -1,16 +1,27 @@
 ---
-title: Getting Started with F#
-description: Find out how to get started with the F# programming language in .NET.
-ms.date: 09/06/2016
+title: Get Started with F#
```
Comment (mairaw) [inline docs/fsharp/get-started/index.md]:
```suggestion
title: Get started with F#
```

File: docs/fsharp/get-started/index.md [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 
-In general, there is no specific way to get started which is better than the rest.  We recommend trying all ways to use F# on your machine to see what you like the best!
+In general, there is no specific that is better than the rest. We recommend trying all ways to use F# on your machine to see what you like the best!
+
+## Get started online
+
+If you'd rather not install F# and .NET on your machine, you can also get started with F# in the browser:
+
+* [Introduction to F# on Azure Notebooks](https://notebooks.azure.com/Microsoft/projects/2018-Intro-FSharp/html/Introduction%20to%20FSharp.ipynb) is a [Jupyter notebook](https://jupyter.org/) on the free Azure Notebooks service. Just sign in with an account and you can hack away directly in the browser!
+* [The Fable REPL](https://fable.io/repl/) is an interactive, in-browser REPL that uses [Fable](https://fable.io/) to translate F# code into JavaScript. Check out the numerous samples that range from F# basics to a fully-fledged video game all executing in your browser!
```
Comment (mairaw) [inline docs/fsharp/get-started/index.md]:
```suggestion
* [The Fable REPL](https://fable.io/repl/) is an interactive, in-browser REPL that uses [Fable](https://fable.io/) to translate F# code into JavaScript. Check out the numerous samples that range from F# basics to a fully fledged video game all executing in your browser!
```

File: docs/fsharp/get-started/index.md [hunk 3]
```diff
@@ -1,16 +1,27 @@
...
 | Linux | N/A | [Get started with VSCode and Ionide](get-started-vscode.md) | [Get started with the .NET Core CLI](get-started-command-line.md) |
 
-In general, there is no specific way to get started which is better than the rest.  We recommend trying all ways to use F# on your machine to see what you like the best!
+In general, there is no specific that is better than the rest. We recommend trying all ways to use F# on your machine to see what you like the best!
+
+## Get started online
+
+If you'd rather not install F# and .NET on your machine, you can also get started with F# in the browser:
+
+* [Introduction to F# on Azure Notebooks](https://notebooks.azure.com/Microsoft/projects/2018-Intro-FSharp/html/Introduction%20to%20FSharp.ipynb) is a [Jupyter notebook](https://jupyter.org/) on the free Azure Notebooks service. Just sign in with an account and you can hack away directly in the browser!
```
Comment (mairaw) [inline docs/fsharp/get-started/index.md]: do you need to specify which kind of account?

File: docs/fsharp/get-started/index.md [hunk 4]
```diff
@@ -1,16 +1,27 @@
...
 ---
-# Getting Started with F# #
+# Get Started with F# #
 
-There are multiple ways to get started with F#.  We have multiple articles written to provide a guide for each major way.  You can use the following table to help in making a decision.
+You can get started with F# on your machine or online.
+
+## Get started on your machine
+
+We have multiple guides on how to install and use F# for the first time on your machine.  You can use the following table to help in making a decision.
```
Comment (mairaw) [inline docs/fsharp/get-started/index.md]:
```suggestion
There are multiple guides on how to install and use F# for the first time on your machine.  You can use the following table to help in making a decision:
```

File: docs/fsharp/get-started/index.md [hunk 5]
```diff
@@ -1,16 +1,27 @@
...
 
 | OS | Prefer Visual Studio | Prefer Visual Studio Code | Prefer a command line |
 | -- |------------------------|--------------------------|-----------------------------|-------------------------|
 | Windows | [Get started with Visual Studio](get-started-visual-studio.md) | [Get started with VSCode and Ionide](get-started-vscode.md) | [Get started with the .NET Core CLI](get-started-command-line.md) |
 | macOS | [Get started with VS for Mac](get-started-with-visual-studio-for-mac.md) | [Get started with VSCode and Ionide](get-started-vscode.md) | [Get started with the .NET Core CLI](get-started-command-line.md) |
 | Linux | N/A | [Get started with VSCode and Ionide](get-started-vscode.md) | [Get started with the .NET Core CLI](get-started-command-line.md) |
```
Comment (mairaw) [inline docs/fsharp/get-started/index.md]: VSCode is the wrong branding. In theory, we should always spell out Visual Studio Code. But if not, at least VS Code separate.

File: docs/fsharp/get-started/index.md [hunk 6]
```diff
@@ -1,16 +1,27 @@
...
+# Get Started with F# #
 
-There are multiple ways to get started with F#.  We have multiple articles written to provide a guide for each major way.  You can use the following table to help in making a decision.
+You can get started with F# on your machine or online.
+
+## Get started on your machine
+
+We have multiple guides on how to install and use F# for the first time on your machine.  You can use the following table to help in making a decision.
 
 | OS | Prefer Visual Studio | Prefer Visual Studio Code | Prefer a command line |
```
Comment (mairaw) [inline docs/fsharp/get-started/index.md]:
```suggestion
| OS | Prefer Visual Studio | Prefer Visual Studio Code | Prefer command line |
```

**Review by mairaw — APPROVED** [review]
LGTM. Left some comments for you to consider.

File: docs/fsharp/get-started/index.md (hunk 3 above)
Comment (cartermp) [reply docs/fsharp/get-started/index.md]: Doesn't need an Azure account, so I'll say a Microsoft account here

**Pull request merged**